megamek/src/megamek/client/bot/princess/BehaviorSettingsFactory.java

```diff
@@ -229,7 +244,7 @@ public boolean saveBehaviorSettings(boolean includeTargets) {
         init(false);
         List<String> names;
         synchronized (behaviorMap) {
-            names = new ArrayList<String>(behaviorMap.keySet());
+            names = new ArrayList<>(behaviorMap.keySet());

```

Comment:
I missed that the declaration was typed, that's my bad for doing a code review of this while at work. All good then, and I have no other objections to the code! I'll hit the approve button momentarily.